numpy/core/code_generators/numpy_api.py

```diff
@@ -350,6 +350,8 @@
     'PyArray_ResolveWritebackIfCopy':       (302,),
     'PyArray_SetWritebackIfCopyBase':       (303,),
     # End 1.14 API
+    'PyArray_KeySort':                      (304,),
+    # End of trial API
```

Comment:
As of now do not think it is too important. Just kept it there, so that if we needed a separate function, it at user level it, then it  might come in handy. Since we have all but zeroed on by parameter , it should be okay to remove it here.

Reply:
Should be removed. We try to err on the side of not changing the C API if possible.